test/tests/ver-1kb/tests

```diff
@@ -0,0 +1,25 @@
+[Tests]
```

Comment:
Please add an exodus test for the case K=10/RT.

You can look at tests specification files in other verification cases to see how to do it.